src/controller/users_controls.py

```diff
@@ -0,0 +1,19 @@
+from fastapi.encoders import jsonable_encoder
+from src.utils.functions import model
+from ..schema.users_model import user
+from ..utils.auth import hash
+    
+async def register(User):
```

Comment:
Add a doc to the function

Reply:
pylint errors would crop up without docstrings